src/bp/core/services/dialog/dialog-engine.ts

```diff
@@ -120,6 +128,52 @@ export class DialogEngine {
     return event
   }
 
+  public changeWorkflow(event: IO.IncomingEvent, nextFlow: string) {
+    const { currentWorkflow, workflows } = event.state.session
+
+    const currentFlowName = currentWorkflow?.replace('.flow.json', '')
+    const parentFlow = this._findFlow(event.botId, `${nextFlow}.flow.json`)?.parent
```

Comment:
possible that this gets undefined ? does it have any impact

Reply:
Hum you're right, it can't be undefined because it throws if it doesn't exist.